### ISSUE TITLE
Class cast exception

### ISSUE THREAD

**Issue by mrbkt** [kestreltechnology/eclipse-kt-advance#9]
Context: in KT Advance perspective, Dovecot project results; modify a C source file by adding blank line (```anvil-connection.c```), receive exception error dialog below along with following stack trace. 

![screen shot 2016-12-31 at 10 32 10 pm](https://cloud.githubusercontent.com/assets/1465947/21580193/fe474618-cfa8-11e6-8716-71788aa3fe64.png)

```{java}
!ENTRY org.eclipse.core.jobs 4 2 2016-12-31 22:29:13.053
!MESSAGE An internal error occurred during: "Analyzing 'anvil-connection.c'".
!STACK 0
java.lang.ClassCastException: com.kt.advance.codan.KTAdvanceMarkerProblemReporter$KTCodanProblemMarker cannot be cast to org.eclipse.cdt.codan.internal.core.model.CodanProblemMarker
	at org.eclipse.cdt.codan.internal.core.model.CodanProblemMarker.equals(CodanProblemMarker.java:263)
	at org.eclipse.cdt.codan.internal.core.model.CodanMarkerProblemReporter.similarMarker(CodanMarkerProblemReporter.java:242)
	at org.eclipse.cdt.codan.internal.core.model.CodanMarkerProblemReporter$2.run(CodanMarkerProblemReporter.java:190)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2240)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2267)
	at org.eclipse.cdt.codan.internal.core.model.CodanMarkerProblemReporter.reconcileMarkers(CodanMarkerProblemReporter.java:185)
	at org.eclipse.cdt.codan.internal.core.model.CodanMarkerProblemReporter.done(CodanMarkerProblemReporter.java:177)
	at org.eclipse.cdt.codan.core.model.AbstractChecker.after(AbstractChecker.java:219)
	at org.eclipse.cdt.codan.internal.core.CodanRunner.processResource(CodanRunner.java:101)
	at org.eclipse.cdt.codan.internal.core.CodanRunner.processResource(CodanRunner.java:58)
	at org.eclipse.cdt.codan.internal.core.CodanBuilder.processResource(CodanBuilder.java:105)
	at org.eclipse.cdt.codan.internal.ui.cxx.CodanPartListener$2.run(CodanPartListener.java:74)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
****
```

**Comment by msiz** [comment]
Must have been fixed with commit 2b63a0c926936b00d1f415a1cd1d273da5d8df0c (checked in within #8 )

**Comment by mrbkt** [comment]
Agreed, no longer an issue
